Tools/autotest/web/index.html

```diff
@@ -5,7 +5,7 @@
 <title>ArduPilot Automatic Testing</title>
 
 <!--CSS -->
-<link rel="stylesheet" href="css/main.css" type="text/css" />
+<link rel="stylesheet" href="/css/main.css" type="text/css" />
```

Comment:
this was deliberately a relative css file to make it easier to test locally. Why change it?

Reply:
I agree that having it relative is better.

But you can see on the autotest page that the css is broken as it cannot find the relative file since previously it was using an absolute path. What is strange is that it still serve the old css file two time...

I don't know how it is done on the server, but we should be able to update the css like the html file. That isn't the case now. You can test this by running a test and looking into the buildlog directory : the index.html is here and update but the css wasn't copied.

Could you confirm the structure on the server ?
It is easy to copy the css directory on same time than the index.html.